### DISCUSSION TITLE
IDAKLUSolver doesn't work on Windows

### DISCUSSION THREAD

**Discussion by RuiheLi** [pybamm-team/PyBaMM#2955] [Show and tell]
PyBaMM: 23.4.1
Python: 3.10.11

Hi @martinjrobins, I am trying to use the IDAKLUSolver on my windows, inside the virtual environment. When I try:
```
import pybamm
model = pybamm.lithium_ion.DFN()
sim = pybamm.Simulation(model,solver=pybamm.IDAKLUSolver(),)
```
It gives errors:
`ImportError: KLU is not installed`
from pybamm\solvers\idaklu_solver.py:113
So I go into idaklu_solver.py and copy the following lines into my jupyter notebook:
```
import casadi
import pybamm
import numpy as np
import numbers
import scipy.sparse as sparse

import importlib

idaklu_spec = importlib.util.find_spec("pybamm.solvers.idaklu")
print(idaklu_spec)
```
and it prints `None`.

I wonder is it because of I am using Windows? As I saw a previous discussion [here](https://github.com/pybamm-team/PyBaMM/discussions/1935)

Thanks a lot!

**Comment by benshang1997** [comment]
Hi @RuiheLi ,Have you solve this issues? I meet the same problem. thank you
